tests/integration/backend_test.py

```diff
@@ -75,16 +75,14 @@
 (requires configuration of Quantinuum username)"
 
 
+@pytest.mark.skipif(skip_remote_tests, reason=REASON)
 @pytest.mark.parametrize("authenticated_quum_backend_qa", [None], indirect=True)
 @pytest.mark.parametrize("language", [Language.QASM, Language.QIR])
 @pytest.mark.timeout(120)
 def test_quantinuum(
     authenticated_quum_backend_qa: QuantinuumBackend, language: Language
 ) -> None:
-    if skip_remote_tests:
-        backend = QuantinuumBackend(device_name="H1-1SC", machine_debug=True)
```

Comment:
We could put in the offline api here?

Reply:
The problem seems to be that the test is parametrized by a "authenticated_quum_backend_qa", which I suppose gets evaluated even if `skip_remote_tests` is `True`. So just changing this line doesn't help.